docs/GraFx-Studio/guides/template-variables/number/index.md

```diff
@@ -0,0 +1,123 @@
+# Number Variables
+
+## Create a Number Variable
+
+Under the Automate icon, click Variables, and add a variable with the "+" sign.
+
+![screenshot](num0.png)
+
+Choose "Number" as the Variable Type.
+
+![screenshot-full](create-numbers.gif)
+
+## Set the Variable Name
+
+Double-click the name, or choose "Rename" under the "..." menu.
+
+![screenshot](num2.png)
+
+## Set General Properties
+
+![screenshot-full](num1.png)
+
+### Visible
+
+Defines if the variable will be visible in the [Studio UI](/GraFx-Studio/concepts/template-management/#studio-ui) to the end user.
+
+### Range
+
+Defines the acceptable range of values to be accepted by the variable placeholder.
+
+### Decimal Settings
+
+#### Decimal Places
+
+Set the number of decimal places that will be displayed.
+
+#### Decimal Character Style
+
+![screenshot-full](ds0.png)
+
+Specify a character style specific for the decimal part of a number.
+
+**Example**
+
+The chosen Paragraph style for the frame is "Text".
+
+![screenshot-full](ds1.png)
+
+A specific character style is defined and has the property to show as "Superscript".
+
+![screenshot-full](ds2.png)
+
+The variable decimal setting shows the decimals will use this character setting for the decimal part.
```

Comment:
This sentence is a bit confusing. I would write it as something like: The decimal character style is set to "Superscript" to use it as character style for the decimal part.

Reply:
Changed it